patches/minecraft/net/minecraft/item/Item.java.patch

```diff
@@ -68,7 +68,7 @@
          CreativeTabs creativetabs = this.func_77640_w();
          return creativetabs != null && (p_194125_1_ == CreativeTabs.field_78027_g || p_194125_1_ == creativetabs);
      }
-@@ -435,11 +445,704 @@
+@@ -435,11 +441,704 @@
```

Comment:
You aren't really changing the contents of this file, so leave it out of the PR.